docs.md

```diff
@@ -267,6 +268,29 @@ $ seml drop [OPTIONS] [PATTERN]
 * `-y, --yes`: Automatically confirm all dialogues with yes.
 * `--help`: Show this message and exit.
 
+## `seml init`
+
+Initialize a new project in the given directory.
+
+**Usage**:
+
+```console
+$ seml init [OPTIONS] [DIRECTORY]
+```
+
+**Arguments**:
+
+* `[DIRECTORY]`: The directory in which to initialize the project.  [default: .]
+
+**Options**:
+
+* `-t, --template TEXT`: The name of the template to use for the project.  [default: default]
```

Comment:
Do we want it as flag or argument? I'd argue for optional argument, seems more natural to me.

Reply:
only one positional argument makes sense to, either directory or template I guess